### DISCUSSION TITLE
Ignore error: Directory already exists.

### DISCUSSION THREAD

**Discussion by viajerus** [cjbarrie/academictwitteR#362] [Q&A]
Hello, 
I am downloading tweets using a for loop and everytime a new loop begins, I get this error: 

```
2: Directory already exists. Existing JSON files may be parsed and returned, choose a new path if this is not intended. 

```
The loop broke after 20 runs and this is the error I received: 

```
Too many errors.
```
Is there a way to ignore the "Directory already exists" error? 

My code: 

```
query <- paste0('(','"COVID19" OR "COVID_19" ',')', ' has:geo')

for (item in res) {
  get_all_tweets(bbox = unname(item),
                 query,
                 start_tweets = "2020-01-01T00:00:00Z",
                 end_tweets = "2022-10-31T00:00:00Z",
                 file = NULL,
                 data_path = tmpdir,
                 export_query = TRUE,
                 bind_tweets = FALSE,
                 is_retweet = FALSE,
                 verbose = TRUE,
                 n = Inf)

  Sys.sleep(60)
  
} 

````
note: res is a list of bounding boxes.

**Comment by chainsawriot** [comment]
First, the message emitted is not an error, but a warning.

https://github.com/cjbarrie/academictwitteR/blob/49480f67c3858ddeea38c62bf890e945c45ec75f/R/utils.R#L145

You can suppress the warning by wrapping the call with `suppressWarnings()`

```r
suppressWarnings({
  get_all_tweets(...)
})
```

However, the "Too many errors" probably has nothing to do with the warnings. It is related to the actual call to the API.

https://github.com/cjbarrie/academictwitteR/blob/bc0df8c85a2470acfa3b06b367ad89c68786acfe/R/utils.R#L6

One possible reason is that there are some malformed things in your `res` so that it breaks at exactly 20. I suggest running the query on-by-on rather than in a loop for a small batch of `item` in your `res` to debug the actual problem.
